web/viewer.css

```diff
@@ -882,8 +882,7 @@ select {
   mask-image: var(--toolbarButton-zoomIn-icon);
 }
 
-#presentationMode::before,
-#secondaryPresentationMode::before {
+#presentationMode::before {
```

Comment:
Here, and for the other button, why are you not just removing this one and keeping the `secondaryPresentationMode` name instead (since the names were intended to reference where the button is placed)?

Reply:
I thought about that but I noticed that the other buttons aren't prefixed with `secondary`: `firstPage`, `lastPage`, `pageRotateCw`, hence I removed `secondary` in order to be consistent.